supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -734,7 +734,7 @@ _Update the `Pod` spec to reflect the changes._
 
 [[spec-file]]
 ==== image:images/yes.png[yes] spec file (noun)
-*Description*: _Spec files_ are used as part of rebuilding RPMs. The spec file outlines how to configure and compile the RPM as well as how to install the files later.
+*Description*: The `rpmbuild` tool uses a _spec file_ to build an RPM package. A spec file provides the necessary information to the build tool by defining instructions in a series of sections.
```

Comment:
The second sentence is pretty heavy on the anthropomorphism. Recommend rephrasing.

Reply:
Thanks, @mportman12 . I'm not sure about this as I'm discussing the relationship between two non-human entities. I often see "provides" used in the context of one component providing something to another component. Would "send" work better?

Reply:
@dfitzmau Is "provides necessary information to the build tool" even needed? The relationship between the spec file and build tool is already defined in the first sentence. How about "A spec file defines instructions in a series of sections"?

Reply:
Thanks. Yes, I can update to state your version of the sentence.